### PR TITLE
Gazebo8 rules

### PR DESCRIPTION
**Pull request by mikaelarguedas**
This adds rules to import gazebo8 for Lunar.

Needed before https://github.com/ros/rosdistro/pull/14564 can be merged

### CODE REVIEW

File: config/gazebo8_debian.upstream.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: gazebo8_debian_import
+method: http://packages.osrfoundation.org/gazebo/debian-stable
+suites: [stretch]
+component: main
+architectures: [amd64, arm64, armhf, i386, source]
+filter_formula: Package (% gazebo-dbg ) | Package (% gazebo ) | Package (% sdformat)
```
Comment (tfoote) [inline config/gazebo8_debian.upstream.yaml]: This probably needs gazebo8 specifically as well as any dependencies that need to come from packages.osrfoundation.org like sdformat? ignition? @j-rivero Can you comment what's needed in this import. Like the gazebo5 versions: https://github.com/ros-infrastructure/reprepro-updater/commit/3c333ac5c6b2fedada768e60582eaa2c437079ae

Comment (j-rivero) [inline config/gazebo8_debian.upstream.yaml]: You are right, we need the usual family of libraries. For `gazebo8`we have [this time](https://bitbucket.org/osrf/gazebo8-release/src/1bd521c4d1a19c6512b461fd1669e77a438ae8d1/ubuntu/debian/control?at=default&fileviewer=file-view-default#control-45):
```
     libsdformat5-dev,
     libignition-math3-dev,
     libignition-transport3-dev (>> 3.0.1)
     libignition-msgs-dev (>= 0.6.999)
```
I think that the rest of the dependencies are available from Ubuntu upstream.

**Comment by mikaelarguedas** [comment]
@j-rivero I updated the rules accordingly. Though I'm not convinced I didnn't miss anything and could use another set of eyes to validate the rules.

**Comment by j-rivero** [comment]
> @j-rivero I updated the rules accordingly. Though I'm not convinced I didnn't miss anything and could use another set of eyes to validate the rules.

I think that we miss some packages, I marked then in bold. This is the whole family summary:

* Gazebo
  * gazebo8 
  * ***gazebo8-plugin-base***
  * ***libgazebo8***
  * ***libgazebo8-dev***
  * ***libgazebo8-dbg***
  * ***(remove gazebo8-dgb)***

* SDFormat
  * libsdformat5
  * libsdformat5-dev
  * libsdformat5-dbg
  * ***sdformat-sdf***

* Ignition math3 (all ok)
   * libignition-math3
   * libignition-math3-dev
   * libignition-math3-dbg

* Ignition msgs (all ok)
   * libignition-msgs0
   * libignition-msgs-dev
   * libignition-msgs0-dbg

* Ignition transport (all ok) 
  * libignition-transport3
  * libignition-transport3-dev 
  * libignition-transport3-dbg

**Comment by mikaelarguedas** [comment]
thanks @j-rivero for the review, updated accrdingly in https://github.com/ros-infrastructure/reprepro-updater/pull/40/commits/b2df359e8aa1686507e259a108772950b759b4ec

**Comment by j-rivero** [comment]
> thanks @j-rivero for the review, updated accrdingly in b2df359

great, looks good to me +1

**Comment by mikaelarguedas** [comment]
thanks @j-rivero.
@tfoote I let you merge this whenever you see fit

**Comment by tfoote** [comment]
I imported into the bootstrap repo, and then into the buildfarm: http://build.ros.org:8080/job/import_upstream/116/console

**Comment by mikaelarguedas** [comment]
Thanks! merging https://github.com/ros/rosdistro/pull/14564 then

**Pull request merged**